layouts/default.vue

```diff
@@ -44,8 +44,7 @@
 </script>
 
 <style lang="scss">
-  html, body {
-    height: 100%;
+  html, body {    height: 100%;
```

Comment:
This styling doesn't seem to look correct anymore.

Reply:
Fixed